### DISCUSSION TITLE
Can't install from F-Droid

### DISCUSSION THREAD

**Discussion by DevDorrejo** [termux/termux-app#2344] [Q&A] [answered]
Phone specs:
Poco F1
android 10, Xiaomi.EU

Can't install termux from F-Droid, but from play store work fine.

![Screenshot_2021-10-14-14-43-13-655_com.miui.packageinstaller.jpg](https://user-images.githubusercontent.com/7697566/137378561-e8576e41-8c9b-40a4-aa3f-725b42b66ebb.jpg)

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app#installation
